packages/google_maps_flutter/google_maps_flutter_web/lib/src/google_maps_flutter_web.dart

```diff
@@ -314,6 +314,6 @@ class GoogleMapsPlugin extends GoogleMapsFlutterPlatform {
 
     onPlatformViewCreated.call(creationId);
 
-    return mapController.widget;
+    return mapController.widget!;
```

Comment:
Why is this safe? Could use a comment.

Reply:
I've added an assert here. `widget` should never be null, unless `dispose` has been called on a map.